lib/_sidebar.erb

```diff
@@ -1,57 +1,35 @@
-    <!-- Sidebar -->
-    <div class="container">
-      <div class="col-sm-2 sidebar">
-        <nav class="nav-sidebar">
-          <ul class="nav tabs">
-          <% if !@site_map['Documents'].nil? %>
+    <aside class="sidebar">
+        <a href="" class="sidebar__close"></a>
+
+        <a href="/" class="logo">
+            <img src="https://minio.io/img/logo/wordmark_white.svg" alt="">
+        </a>
+
+        <ul class="doc-link">
             <%- @site_map['Documents'].each do |categories| -%>
-             <%- categories.each do |key, value| -%>
-              <%= key -%>
-              <%- first_entry = true -%>
-                <%- value.each do |key2| -%>
-                      <%- if key===@site_map['Documents'][0].keys[0] && first_entry -%>
-                        <%- active_keyword="active" -%>
-                        <%- first_entry = false -%>
-                      <%- else -%>
-                        <%- active_keyword="" -%>
-                      <%- end -%>
-                      <%- if reveal(raw(key2["Link"])) -%>
-                        <li>
-                          <%- if !key2["Slug"] -%>
-                            <%- key2["Slug"] = get_slug(key2) -%>
-                          <%- end -%>
-                          <a class="<%=active_keyword%>" href="#<%=key2["Slug"]%>" data-toggle="tab">
-                            <%= key2.keys[0] -%>
-                          </a>
-                        </li>
-                      <%- end -%>
+
+                <%- categories.each do |key, value| -%>
+                    <li class="doc-link__category">
+                        <a href="" class="doc-link__header"><%= key -%></a>
+
+                        <%- first_entry = true -%>
+
+                        <ul class="doc-link__items">
+                            <%- value.each do |key2| -%>
+                                <%- if key===@site_map.keys[0] && first_entry -%>
+                                    <%- active_keyword="active" -%>
+                                    <%- first_entry = false -%>
+                                <%- else -%>
+                                    <%- active_keyword="" -%>
+                                <%- end -%>
+                                <li><a class="<%=active_keyword%>" href="#<%=key2["Slug"]%>"><%=key2.keys[0]%></a></li>
```

Comment:
We would have to check for `reveal(raw(key2["Link"]))` to be true to prevent documents that have broken links to appear on the sidebar.